evalml/tuners/tuner.py

```diff
@@ -89,6 +89,36 @@ def _convert_to_pipeline_parameters(self, flat_parameters):
             pipeline_parameters[component_name][parameter_name] = parameter_value
         return pipeline_parameters
 
+    def get_starting_parameters(self, hyperparameter_ranges, random_seed=0):
```

Comment:
Add the `hyperparam_ranges` and `random_seed` args to limit the starting parameters to only custom hyperparameters that users pass in.